2012-12-18 14:17:14 Error: [MissingControllerException] Controller class CssController could not be found.
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2012-12-18 15:03:52 Error: [MissingControllerException] Controller class CssController could not be found.
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2012-12-30 19:51:28 Error: [MissingControllerException] Controller class OndeController could not be found.
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-01 01:30:54 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:30:54 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-01 01:31:00 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:31:00 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-01 01:31:02 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:31:02 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-01 01:31:02 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:31:02 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-01 01:31:03 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:31:03 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-01 01:31:05 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [/Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Elements/menu.ctp, line 3]
2013-01-01 01:31:05 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Error/ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/Applications/X...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/Applications/X...', 3, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/Applications/X...', 3, Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 18:29:38 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
#0 C:\xampp\htdocs\moti\lib\Cake\Model\Datasource\DboSource.php(261): Mysql->connect()
#1 C:\xampp\htdocs\moti\lib\Cake\Model\ConnectionManager.php(101): DboSource->__construct(Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Model\Model.php(3170): ConnectionManager::getDataSource('default')
#3 C:\xampp\htdocs\moti\lib\Cake\Model\Model.php(1096): Model->setDataSource('default')
#4 C:\xampp\htdocs\moti\lib\Cake\Model\Model.php(3192): Model->setSource('inscricoes')
#5 C:\xampp\htdocs\moti\lib\Cake\Model\Model.php(1305): Model->getDataSource()
#6 C:\xampp\htdocs\moti\lib\Cake\View\Helper\FormHelper.php(197): Model->schema()
#7 C:\xampp\htdocs\moti\lib\Cake\View\Helper\FormHelper.php(443): FormHelper->_introspectModel('Inscricao', 'fields')
#8 C:\xampp\htdocs\moti\app\View\Inscricoes\inscrever.ctp(3): FormHelper->create('Inscricao')
#9 C:\xampp\htdocs\moti\lib\Cake\View\View.php(923): include('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\moti\lib\Cake\View\View.php(886): View->_evaluate('C:\xampp\htdocs...', Array)
#11 C:\xampp\htdocs\moti\lib\Cake\View\View.php(476): View->_render('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(956): View->render(NULL, NULL)
#13 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#14 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(InscricoesController), Object(CakeRequest), Object(CakeResponse))
#15 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#16 {main}
2013-01-08 19:24:33 Error: Fatal Error (4): syntax error, unexpected '{' in [C:\xampp\htdocs\moti\app\Controller\InscricoesController.php, line 10]
2013-01-08 19:24:33 Error: [FatalErrorException] syntax error, unexpected '{'
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 19:24:48 Error: Fatal Error (4): syntax error, unexpected '{' in [C:\xampp\htdocs\moti\app\Controller\InscricoesController.php, line 10]
2013-01-08 19:24:48 Error: [FatalErrorException] syntax error, unexpected '{'
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 19:34:32 Error: [MissingControllerException] Controller class InscricaoController could not be found.
#0 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-09 20:34:49 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [C:\xampp\htdocs\moti\app\View\Inscricoes\inscrever.ctp, line 4]
2013-01-09 20:34:49 Error: [FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 4)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 4, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 4, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-13 19:15:55 Error: [MissingTableException] Table inscricoes for model Inscricao was not found in datasource default.
#0 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Model/Model.php(3192): Model->setSource('inscricoes')
#1 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Model/Model.php(1305): Model->getDataSource()
#2 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/View/Helper/FormHelper.php(197): Model->schema()
#3 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/View/Helper/FormHelper.php(443): FormHelper->_introspectModel('Inscricao', 'fields')
#4 /Applications/XAMPP/xamppfiles/htdocs/moti/app/View/Inscricoes/inscrever.ctp(3): FormHelper->create('Inscricao', Array)
#5 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/View/View.php(923): include('/Applications/X...')
#6 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/View/View.php(886): View->_evaluate('/Applications/X...', Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/View/View.php(476): View->_render('/Applications/X...')
#8 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Controller/Controller.php(956): View->render(NULL, NULL)
#9 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Routing/Dispatcher.php(193): Controller->render()
#10 /Applications/XAMPP/xamppfiles/htdocs/moti/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(InscricoesController), Object(CakeRequest), Object(CakeResponse))
#11 /Applications/XAMPP/xamppfiles/htdocs/moti/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-01-16 16:50:45 Error: [MissingControllerException] Controller class PalestranteController could not be found.
#0 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-16 17:02:11 Error: [MissingControllerException] Controller class PalestranteController could not be found.
#0 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-16 17:53:31 Error: [MissingViewException] View file "C:\xampp\htdocs\moti\app\View\Palestras\index.ctp" is missing.
#0 C:\xampp\htdocs\moti\lib\Cake\View\View.php(473): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(956): View->render(NULL, NULL)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrasController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 18:06:32 Error: Fatal Error (4): syntax error, unexpected end of file in [C:\xampp\htdocs\moti\app\Controller\PalestrasController.php, line 24]
2013-01-16 18:06:32 Error: [FatalErrorException] syntax error, unexpected end of file
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 24, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 24, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 18:11:35 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\moti\app\Controller\PalestrantesController.php, line 5]
2013-01-16 18:11:35 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 18:11:49 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\moti\app\Controller\PalestrantesController.php, line 5]
2013-01-16 18:11:49 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 18:12:27 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\moti\app\Controller\PalestrantesController.php, line 5]
2013-01-16 18:12:27 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 18:14:12 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\moti\app\Controller\PalestrantesController.php, line 5]
2013-01-16 18:14:12 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 19:28:11 Error: Fatal Error (4): syntax error, unexpected ''telefone'' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [C:\xampp\htdocs\moti\app\Model\Inscricao.php, line 11]
2013-01-16 19:28:11 Error: [FatalErrorException] syntax error, unexpected ''telefone'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 11)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 11, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 11, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 20:19:00 Error: Fatal Error (4): syntax error, unexpected ';', expecting ')' in [C:\xampp\htdocs\moti\app\Model\Inscricao.php, line 19]
2013-01-16 20:19:00 Error: [FatalErrorException] syntax error, unexpected ';', expecting ')'
#0 C:\xampp\htdocs\moti\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 19)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 19, Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 19, Array)
#3 C:\xampp\htdocs\moti\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-16 21:34:30 Error: [MissingActionException] Action PalestrantesController::joao-silva() could not be found.
#0 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-16 21:34:41 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:36:16 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:39:59 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:40:14 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:47:09 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:47:24 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-16 21:48:16 Error: [NotFoundException] Palestrante invalido
#0 [internal function]: PalestrantesController->view()
#1 C:\xampp\htdocs\moti\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(PalestrantesController), Array)
#2 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\moti\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PalestrantesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\moti\app\webroot\index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
